src/go/k8s/controllers/redpanda/console_controller_test.go

```diff
@@ -193,13 +193,20 @@ var _ = Describe("Console controller", func() {
 	Context("When updating Console", func() {
 		ctx := context.Background()
 		It("Should not create new ConfigMap if no change on spec", func() {
+			var configmapNsn string
+
 			By("Getting Console")
 			consoleLookupKey := types.NamespacedName{Name: ConsoleName, Namespace: ConsoleNamespace}
-			createdConsole := &redpandav1alpha1.Console{}
-			Expect(k8sClient.Get(ctx, consoleLookupKey, createdConsole)).Should(Succeed())
+			Eventually(func() bool {
+				createdConsole := &redpandav1alpha1.Console{}
+				Expect(k8sClient.Get(ctx, consoleLookupKey, createdConsole)).Should(Succeed())
```

Comment:
Please don't mix the `Eventually` with `Expect`

Reply:
```suggestion
			Eventually(func() bool {
				createdConsole := &redpandav1alpha1.Console{}
				if err := k8sClient.Get(ctx, consoleLookupKey, createdConsole); err != nil {
					fmt.Fprintf(GinkgoWriter, "failed to get %s: %v\n", consoleLookupKey.Name, err)
					return false
				}	
```

Reply:
Sorry, copy pasta updated